### ISSUE TITLE
[Screen Reader - Open and close New Tab]: Not able to open or close any tab using 'Ctrl+Shift+T' or 'Ctrl+Shift+W' when narrator focus is not present in the edit area.

### ISSUE THREAD

**Issue by Chandresh1007** [microsoft/terminal#8480]
**User Experience:**
If any application does not allow assistive technology to programmatically execute available actions on objects it would be difficult for the platforms that provide Accessibility APIs, and would in turn affect the screen reader dependent users to determine the programmatically executable actions on objects.

**Test Environment:**
OS: Win 10 2004 Build (20262.1010)
App: Windows Terminal
App version: 1.4.3243.0

**Repro Steps:**
1. Open Windows terminal app.
2. Launch Narrator.
3. Turn on the scan mode using 'Caps+Space'.
4. Navigate the narrator focus out of edit area.
5. Now press 'Ctrl+Shift+T' or 'Ctrl+Shift+W' to open or close any tab.
6. Observe the issue.

**Actual Result:**
**Not able to open or close any tab using 'Ctrl+Shift+T' or 'Ctrl+Shift+W' when narrator focus is not present in the edit area** in the scan mode of the narrator.

**Expected Result:**
User **should be able to open or close any tab using 'Ctrl+Shift+T' or 'Ctrl+Shift+W'** even when narrator focus is not present in the edit area.

**Note:** NVDA browse mode not working.

**MAS Reference:**
**MAS4.2.9**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Baa4c94fe-60fa-4003-8a52-7abca179bde2%7D&cid=f00419a4-9d8d-4a89-a0e0-d39e9cb77c27

[MAS4.2.9_Not able to open or close any tab when focus is not in edit area.zip](https://github.com/microsoft/terminal/files/5636124/MAS4.2.9_Not.able.to.open.or.close.any.tab.when.focus.is.not.in.edit.area.zip)

**Comment by zadjii-msft** [comment]
Yea, this is probably more fundamental than just #8462 or #6679 - our keybindings are only bound within the context of Terminal Controls. They aren't bound anywhere else in the app, mostly because the entire app is just Terminal Controls. 

I know @DHowett has had a mind to change this in the past, but changing this is not a minor bugfix - it would be a pretty fundamental refactoring of our key handling.

**Comment by zadjii-msft** [comment]
stashed my prototype in `dev/migrie/b/8480-keybindings-in-tabs`

**Comment by carlos-zamora** [comment]
@Chandresh1007 This is now fixed in v1.12.10334.0. Can you please confirm?

**Comment by zadjii-msft** [comment]
@v-rpundir This one too 😉

**Comment by v-rpundir** [comment]
Closing as fixed on Windows Terminal Preview Version: 1.13.10336.0

**Comment by zadjii-msft** [comment]
@v-rpundir Thanks! There are a couple others that should be fixed in this release as well: https://github.com/microsoft/terminal/issues?q=is%3Aopen+label%3AResolution-Fix-Committed+label%3AA11yWCAG

Not sure what the cleanest way to ping you about these all is 😅

**Comment by v-rpundir** [comment]
Hi @zadjii-msft Just simply mention me in individual bug or in one bug with other bugs query with below details:
1. Fix Terminal Version
2. Fix Terminal build download link

Thanks :):
